common/src/main/java/tachyon/network/protocol/RPCBlockReadRequest.java

```diff
@@ -27,11 +27,15 @@
   private final long mBlockId;
   private final long mOffset;
   private final long mLength;
+  private final long mLockId;
+  private final long mSessionId;
 
-  public RPCBlockReadRequest(long blockId, long offset, long length) {
+  public RPCBlockReadRequest(long blockId, long offset, long length, long lockId, long sessionId) {
```

Comment:
All the RPC related classes don't have javadoc (essentially all the subclasses of EncodedMessage). I think it would make more sense to create a ticket to track and write them separately.